src/resources/assets/js/components/enso/bulma/BooleanFilter.vue

```diff
@@ -2,6 +2,7 @@
 
     <vue-filter
             :options="options"
+            :icons="true"
             v-bind="$attrs"
             v-on="$listeners">
```

Comment:
... this line will cascade the `@input` event, which the parent will use to update the `v-model` ... (continued)